src/lightning-address.ts

```diff
@@ -47,7 +50,7 @@ export default class LightningAddress {
     const result = await fetch(`${this.options.proxy}?${new URLSearchParams({ ln: this.address }).toString()}`);
     const json = await result.json();
     this.lnurlpData = json.lnurlp;
-    this.keysendData = json.keysend;
+    this.keysendData = parseKeysendResponse(json.keysend);
```

Comment:
if we do this here for the keysend params, then lets also do it for the lnurl params

Reply:
Yes, we are doing it the same way (now that we merged #13)
https://github.com/getAlby/alby-tools/blob/0134d3718efd51652e0a44a2f9b3cfbd13436a93/src/lightning-address.ts#L53
https://github.com/getAlby/alby-tools/blob/0134d3718efd51652e0a44a2f9b3cfbd13436a93/src/lightning-address.ts#L60